components/python/pyparsing-35/manifests/sample-manifest.p5m

```diff
@@ -0,0 +1,39 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"), version 1.0.
+# You may only use this file in accordance with the terms of version
+# 1.0 of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source.  A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2020 <contributor>
+#
+
+set name=pkg.fmri value=pkg:/$(COMPONENT_FMRI)@$(IPS_COMPONENT_VERSION),$(BUILD_VERSION)
+set name=pkg.summary value="$(COMPONENT_SUMMARY)"
+set name=info.classification value="$(COMPONENT_CLASSIFICATION)"
+set name=info.upstream-url value=$(COMPONENT_PROJECT_URL)
+set name=info.source-url value=$(COMPONENT_ARCHIVE_URL)
+set name=org.opensolaris.consolidation value=$(CONSOLIDATION)
+
+license $(COMPONENT_LICENSE_FILE) license='$(COMPONENT_LICENSE)'
+
+file path=usr/lib/python3.5/vendor-packages/pyparsing-$(COMPONENT_VERSION)-py3.5.egg-info/PKG-INFO
+file path=usr/lib/python3.5/vendor-packages/pyparsing-$(COMPONENT_VERSION)-py3.5.egg-info/SOURCES.txt
+file path=usr/lib/python3.5/vendor-packages/pyparsing-$(COMPONENT_VERSION)-py3.5.egg-info/dependency_links.txt
+file path=usr/lib/python3.5/vendor-packages/pyparsing-$(COMPONENT_VERSION)-py3.5.egg-info/top_level.txt
+file path=usr/lib/python3.5/vendor-packages/pyparsing.py
+file path=usr/lib/python3.7/vendor-packages/pyparsing-$(COMPONENT_VERSION)-py3.7.egg-info/PKG-INFO
+file path=usr/lib/python3.7/vendor-packages/pyparsing-$(COMPONENT_VERSION)-py3.7.egg-info/SOURCES.txt
+file path=usr/lib/python3.7/vendor-packages/pyparsing-$(COMPONENT_VERSION)-py3.7.egg-info/dependency_links.txt
+file path=usr/lib/python3.7/vendor-packages/pyparsing-$(COMPONENT_VERSION)-py3.7.egg-info/top_level.txt
+file path=usr/lib/python3.7/vendor-packages/pyparsing.py
+file path=usr/lib/python3.9/vendor-packages/pyparsing-$(COMPONENT_VERSION)-py3.9.egg-info/PKG-INFO
+file path=usr/lib/python3.9/vendor-packages/pyparsing-$(COMPONENT_VERSION)-py3.9.egg-info/SOURCES.txt
+file path=usr/lib/python3.9/vendor-packages/pyparsing-$(COMPONENT_VERSION)-py3.9.egg-info/dependency_links.txt
+file path=usr/lib/python3.9/vendor-packages/pyparsing-$(COMPONENT_VERSION)-py3.9.egg-info/top_level.txt
+file path=usr/lib/python3.9/vendor-packages/pyparsing.py
```

Comment:
You'll need to regenerate the sample-manifest because here aren't versions for python-3.7 and 3.9.